DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: metapredict
 Type: Package
 Title: Meta-analysis of gene expression using the elastic net
-Version: 1.1.0
+Version: 1.1.1
```

Comment:
devtools::check says Depends should be 3.5 instead of 3.2, based on serialized objects. Do you see this as well?